### PR TITLE
Fix CampusTileSource

### PR DESCRIPTION
**Pull request by jadtl**
A small fix to the campus tile source so that it picks a random server at every request, the bound for the random number is also fixed.

### CODE REVIEW

File: app/src/main/java/com/github/polypoly/app/MapActivity.kt [hunk 1]
```diff
@@ -216,9 +216,9 @@ class MapActivity : ComponentActivity() {
     }
 
 
-    class CampusTileSource(private val serverId: Int, private val floorId: Int) : OnlineTileSourceBase("EPFLCampusTileSource", 0, 18, 256, ".png", arrayOf()) {
+    class CampusTileSource(private val floorId: Int) : OnlineTileSourceBase("EPFLCampusTileSource", 0, 18, 256, ".png", arrayOf()) {
         override fun getTileURLString(pMapTileIndex: Long): String {
-            return "https://plan-epfl-tiles$serverId.epfl.ch/1.0.0/batiments/default/20160712/$floorId/3857/${MapTileIndex.getZoom(pMapTileIndex)}/${MapTileIndex.getY(pMapTileIndex)}/${MapTileIndex.getX(pMapTileIndex)}.png"
+            return "https://plan-epfl-tiles${Random.nextInt(3)}.epfl.ch/1.0.0/batiments/default/20160712/$floorId/3857/${MapTileIndex.getZoom(pMapTileIndex)}/${MapTileIndex.getY(pMapTileIndex)}/${MapTileIndex.getX(pMapTileIndex)}.png"
```
Comment (vrooomer) [inline app/src/main/java/com/github/polypoly/app/MapActivity.kt:221]: Maybe putting a 3 here is a bad thing as it creates a magic number?

**Comment by vrooomer** [comment]
Perfect, thanks! I found a small issue, I let you check this before the merge.

~ v:)

**Review by vrooomer — CHANGES_REQUESTED** [review]
Perfect, thanks! I found a small issue, I let you check this before the merge.

~ v:)

File: app/src/main/java/com/github/polypoly/app/MapActivity.kt (hunk 1 above)
Comment (jadtl) [reply app/src/main/java/com/github/polypoly/app/MapActivity.kt:221]: You're right, and it's not clear why it is 3, I'll change that, thanks.

**Comment by jadtl** [comment]
Is it better like this?

**Review by vrooomer — APPROVED** [review]
It doesn't fix it but I guess for the usage of it, it is acceptable :)

**Pull request merged**